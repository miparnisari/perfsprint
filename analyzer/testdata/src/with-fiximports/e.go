package a

import (
	"fmt" // want "fiximports: Fix imports"
)

func fmte() {
	// fmt removed as only import
	println("Hello, World!", fmt.Sprintf("%s", "toto")) // want "string-format: fmt.Sprintf can be replaced with just using the string"
}
