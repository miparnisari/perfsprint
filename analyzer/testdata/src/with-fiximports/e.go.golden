package a

import (
	// want "fiximports: Fix imports"
)

func fmte() {
	// fmt removed as only import
	println("Hello, World!", "toto") // want "string-format: fmt.Sprintf can be replaced with just using the string"
}
